js-extensions/src/ResourceLoadTracker.js

```diff
@@ -118,18 +133,28 @@ export default class ResourceLoadTracker {
             }
         }
     }
-    
-    _addCSS(url) {
+
+    _addCSS(url, onload) {
         const cssURLPrefix = jsModules.getAdjunctURL();
-        jsModules.addCSSToPage(cssURLPrefix + '/' + url);
+        const cssURL = cssURLPrefix + '/' + url;
+
+        jsModules.addCSSToPage(cssURL);
+
+        const linkElId = jsModules.toCSSId(cssURL);
+        const linkEl = document.getElementById(linkElId);
+        if (linkEl) {
+            linkEl.onload = onload;
+        } else {
+            onload();
+        }
```

Comment:
@NicuPascu I just added the above check on `linkEl`. That code is not in the version o `js-extensions` that we published.